docs/src/views/ShorthandProps.tsx

```diff
@@ -1,37 +1,263 @@
 import * as React from 'react'
 import { NavLink } from 'react-router-dom'
+import { Header } from 'semantic-ui-react'
 import DocPage from '../components/DocPage/DocPage'
 import CodeSnippet from '../components/CodeSnippet'
 import Button from '../../../src/components/Button/Button'
 
+const joinToString = stringOrArray => {
+  if (typeof stringOrArray === 'string') {
+    return stringOrArray
+  }
+
+  return stringOrArray.join('\n')
+}
+
+const header = content => <Header as="h2">{content}</Header>
+const subheader = content => <Header as="h3">{content}</Header>
+
+const code = content => <code>{content}</code>
+const codeExample = snippet => <CodeSnippet value={joinToString(snippet)} />
+
 class ShorthandProps extends React.Component {
   render() {
     return (
       <DocPage title="Shorthand Props">
         <p>
-          Most components support two APIs: <code>shorthand</code> and <code>children</code>. With
-          the <code>shorthand</code> definition we are adding some props to the component in order
-          to define it’s content and behavior.
+          It is quite common for Stardust component to have "slots" which accept shorthand values.
+          For example, {code('Button')} component has an {code('icon')} slot which value defines the
+          icon that will be rendered.
+        </p>
+
+        {codeExample([`<Button icon='chess rook' />`])}
+
+        <p>
+          So, what this string value (i.e. {code("'chess rook'")}) means for {code('Button')}{' '}
+          component and, essentially, how it transforms into rendered {code('<Icon />')} element?
```

Comment:
In my opinion, this question doesn't bring any value to the user. Let's try to remove it and maybe provide bullet points for explaining this points below.